packages/mui-system/src/createTheme/createTheme.d.ts

```diff
@@ -34,12 +39,12 @@ export interface Theme extends CssContainerQueries {
   breakpoints: Breakpoints;
   direction: Direction;
   palette: Record<string, any> & { mode: 'light' | 'dark' };
-  shadows?: unknown;
+  shadows?: Shadows;
   spacing: Spacing;
-  transitions?: unknown;
+  transitions?: Transitions;
   components?: Record<string, any>;
-  mixins?: unknown;
-  typography?: unknown;
+  mixins?: Mixins;
+  typography?: Typography;
   zIndex?: unknown;
```

Comment:
The only `unknown` left is `zIndex`. Is there a reason not to type it the same way as `ThemeOptions`?

Reply:
my bad, fixed 🙏